docs/contributor/releasing.md

```diff
@@ -0,0 +1,107 @@
+   Create a PR to `opentelemetry-collector-components/{RELEASE_BRANCH}` with the following changes:
+   - `sec-scanners-config.yaml`:  
+     Update the tag of the `kyma-otel-collector` image with the new module version following the `OTEL_VERSION-OCC_VERSION` pattern. For example, `europe-docker.pkg.dev/kyma-project/prod/kyma-otel-collector:0.102.1-0.0.1`.
+
+6. Merge the PR.
+
+
+7. To make sure that the release tags point to the HEAD commit of the `opentelemetry-collector-components/{RELEASE_BRANCH}` branch, rebase the upstream branch into the local branch after the merge was successful.
+
+   ```bash
+   git rebase upstream/{RELEASE_BRANCH} {RELEASE_BRANCH}
+   ```
+   
+7. Create tags for every go module in this repository.
+   For every module in receiver, processor, exporter, and extension directories, create a tag with the new version.
+
+   ```bash
+   git tag {RELATIVE_MODULE_PATH}/v{RELEASE_VERSION}
+   # eg. git tag receiver/dummymetricsreceiver/v1.0.0
+   ```
+
+   The create-and-push-tags target in the Makefile helps to create and push tags for all modules.
+   ```bash
+   OCC_VERSION={RELEASE_VERSION} REMOTE={REPOSITORY_REMOTE} make create-and-push-tags
+   # eg. OCC_VERSION=1.0.0 REMOTE=upstream make create-and-push-tags
+   ```
+
+8. In the `opentelemetry-collector-components/{RELEASE_BRANCH}` branch, create release tags for the HEAD commit.
+
+   ```bash
+   git tag v{RELEASE_VERSION}
+   ```
+
+9. Push the tag to the upstream repository.
+
+   ```bash
+   git push {REPOSITORY_REMOTE} v{RELEASE_VERSION}
+   ```
+
+   The {RELEASE_VERSION} tag triggers a GitHub action (`GitHub Release`). 
+
+10. Verify the [status](https://github.com/kyma-project/opentelemetry-collector-components/actions) of the GitHub action (`GitHub Release`).
+    - Once the GitHub action succeed, the new GitHub release is available under [releases](https://github.com/kyma-project/opentelemetry-collector-components/releases).
+    - If the GitHub action fails, re-trigger it by removing the {RELEASE_VERSION} tag from upstream and pushing it again:
+
+      ```bash
+      git push --delete upstream v{RELEASE_VERSION}
+      git push upstream v{RELEASE_VERSION}
+      ```
+
+11. If the previous release was a bugfix version (patch release) that contains cherry-picked changes, these changes might appear again in the generated change log. If there are redundant entries, edit the release description and remove them.
+
+## Changelog
+
+Every PR's title must adhere to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification for an automatic changelog generation. It is enforced by a [semantic-pull-request](https://github.com/marketplace/actions/semantic-pull-request) GitHub Action.
+
+### Pull Request Title
+
+Due to the Squash merge GitHub Workflow, each PR results in a single commit after merging into the main development branch. The PR's title becomes the commit message and must adhere to the template:
```

Comment:
```suggestion
Because of the squash-and-merge GitHub workflow, each PR results in a single commit after merging into the main development branch. The PR's title becomes the commit message and must adhere to the template:
```